husky_components/dropdown/main.js

```diff
@@ -291,7 +337,25 @@ define([], function() {
         },
 
         // hide dropDown
-        hideDropDown: function() {
+        hideDropDown: function(event) {
+            // ignore it if the target is children of trigger but not children of item
+            if (!!event
+                && ((
+                    $(event.target).hasClass('husky-dropdown-trigger')
+                    || $(event.target).parents().hasClass('husky-dropdown-trigger')
+                )
+                && !(
+                    $(event.target).hasClass('husky-dropdown-item')
+                    || $(event.target).parents().hasClass('husky-dropdown-item')
+                )
```

Comment:
Can't the previous two `&&` be connected to one? Maybe some filter that checks if the element has one class and doesn't have another one? Maybe also the two next statements can somehow be included like this.